notebooks/growth_model_forecast.ipynb

```diff
@@ -0,0 +1,1186 @@
+{
```

Comment:
<p>Let's update this to something like:</p><p><code>In this notebook, we will be developing a time series forecasting model i.e. ARIMA model (add hyperlink for ARIMA: https://en.wikipedia.org/wiki/Autoregressive_integrated_moving_average) to predict customer/user growth that enterprises can leverage to help improve their businesses. We will be using a publicly available monthly revenue dataset from Kaggle (add the link for dataset).</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/customer-growth-model/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>